.gitignore

```diff
@@ -1,3 +1,5 @@
+.idea/
+
```

Comment:
This should probably be part of your global gitignore: https://stackoverflow.com/questions/7335420/global-git-ignore

Reply:
@clue Ok. I have reverted the gitignore code. Can you please recheck?

Reply:
Looks like your last commit introduced some additional changes? Can you revert those and squash your changes into a single commit?